lib/properties/plug-energy-monitor/current-monitor-property.js

```diff
@@ -17,6 +17,10 @@ class CurrentMonitorProperty extends TuyaProperty {
     value = value/1000.0; //plug sends current in unit mA
     this.setCachedValueAndNotify(value);
   }
+
+  autodps(obj) {
```

Comment:
I set the autodps intentionally not enabled because it would set the dps slot 9 (outputs a static 0) as the current-property (first Property after on/off). Just tested by clearing the config and now autodps configures: `{"dps":{"on":1,"currentMonitor":9,"powerMonitor":18,"voltageMonitor":19},"sockets":1}` which is unfortunately wrong now :)

DPS: 
```
dps: {
   '1': false,  	//on/off
   '9': 0,       	//static 0/unknown
   '18': 0,	        //electric current  (mA)
   '19': 0, 	        //power watt * 10
   '20': 2303,          // voltage in volt * 10
   '21': 1,
   '22': 715,
   '23': 30333,
   '24': 20561,
   '25': 1010
 }
```

Reply:
Ah ok, the plug I tested it with does not seem to have the phantom property 9, therefore it worked properly for me using autodps as well